code/messagingtestutils/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("com.android.application")
+}
+
+val mavenCoreVersion: String by project
+
+android {
+      namespace = "com.adobe.marketing.mobile.messagingtestutils"
+
+    defaultConfig {
+        compileSdk = BuildConstants.Versions.COMPILE_SDK_VERSION
+        minSdk = BuildConstants.Versions.MIN_SDK_VERSION
+        targetSdk = BuildConstants.Versions.TARGET_SDK_VERSION
+        versionCode = BuildConstants.Versions.VERSION_CODE
+        versionName = BuildConstants.Versions.VERSION_NAME
+    }
+
+    buildTypes {
+        getByName(BuildConstants.BuildTypes.RELEASE)  {
+            isMinifyEnabled = false
+        }
+    }
+}
+
+dependencies {
+    implementation(project(":messaging"))
+    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion-SNAPSHOT")
+
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.12.7")
+    implementation("androidx.test.ext:junit:1.1.5")
+
+    implementation("androidx.appcompat:appcompat:1.6.1")
+    implementation("androidx.constraintlayout:constraintlayout:2.1.4")
```

Comment:
Do we use appcompat and constraintlayout in messagingtestutils? Looks like these need to be in app gradle file

Reply:
That slipped in.. We dont need that